tests/e2e/fixtures/storage_efs_dependencies.py

```diff
@@ -287,6 +287,7 @@ def on_delete():
 
 @pytest.fixture(scope="class")
 def dynamic_provisioning(metadata, region, request, cluster):
+    associate_iam_oidc_provider(cluster, region)
```

Comment:
I think we should just add `--with-oidc` here: https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/fixtures/cluster.py#L22

Reply:
we are running the test with this fix now, let's change this in next release